pkg/sql/schemachanger/scbuild/internal/scbuildstmt/alter_table_alter_primary_key.go

```diff
@@ -528,6 +536,15 @@ func makeShardedDescriptor(b BuildCtx, t alterPrimaryKeySpec) *catpb.ShardedDesc
 	}
 }
 
+// recreateAllSecondaryIndexes recreates all secondary indexes. While the key
+// columns remain the same in the face of a primary key change, the key suffix
+// columns or the stored columns may not.
+func recreateAllSecondaryIndexes(
+	b BuildCtx, tbl *scpb.Table, newPrimaryIndex, sourcePrimaryIndex *scpb.PrimaryIndex,
+) {
+	// TODO(postamar): implement in 23.1
+}
```

Comment:
doesn't this need to return a not implemented error?

Reply:
No. I added a comment to clarify.